### PR TITLE
Post: Organising a Crystal conference in 2020

### PR DESCRIPTION
**Pull request by lbarasti**
image

### CODE REVIEW

File: _posts/2020-11-18-organising-raw-crystal.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Picking a time for the conference was quite challenging: on one hand, I wanted to make the event accessible to as many people across the world as possible. On the other hand, it is impossible to make everyone happy on a global scale.
+
+I decided to make a choice based on the following constraints:
+* the selected schedule should be friendly to people based in Argentina - that's where many members of the core team are based
+* the selected schedule should not affect my circadian rhythm too badly - I'm based in London, UK
+* the total span of the event should be manageable for a single person to run - so no marathons
+
+I also reviewed geo data related to users of my personal blog and to users of my YouTube channel. This reminded me that Crystal has a significant number of friends in North America.
+
+Based on all the above, I decided to go for a 6-hour-long conference starting at 8 am PST.
```
Comment (straight-shoota) [inline _posts/2020-11-18-organising-raw-crystal.md:45]: I'd strongly suggest to communicate times (exclusively) in UTC. Everyone should easily now their respective UTC offset, but not the offset of some random other time zone (PST).
```suggestion
Based on all the above, I decided to go for a 6-hour-long conference starting at 16:00 UTC.
```

**Review by straight-shoota — COMMENTED** [review]
You're doing a really awesome job! Your texts are very nicely written and enjoyable to read. I'm glad you've volunteered to organize Raw Crystal 2020.

File: _posts/2020-11-18-organising-raw-crystal.md (hunk 1 above)
Comment (straight-shoota) [reply _posts/2020-11-18-organising-raw-crystal.md:45]: (Optionally, a supplementary list of the observed time in other selected locations (like those mentioned elsewhere in the article) could be convenient)

**Comment by bcardiff** [comment]
Merged at 37fa6fdcab77554c0be0945abc77fe5dac156d8d (I updated the date...). Thanks @lbarasti

**Comment by straight-shoota** [comment]
travis-ci doesn't process this https://travis-ci.org/github/crystal-lang/crystal-website/builds/744669328

**Comment by bcardiff** [comment]
Screen Shot 2020-11-19 at 11 22 22

There are some serious queue delays, but the build are going through, eventually.

**Pull request closed without merging**